README.md

```diff
@@ -1 +1,20 @@
-# ECLDoc
\ No newline at end of file
+# ECLDoc
+
+Ecldoc is a tool for generating API Documentation for ECL Project based on their docstrings. The executable ecldoc is the main program that parses the sources and generate Documentation.
+
+## Requirements
+
+- You need Python3
+- Install pip3, setuptools
+- Install pdflatex (sudo apt-get install texlive-full)
+
+## Get started
+
+If you are on Windows, please make sure to follow [windows guidelines](docs/windows.md#windows) too.
+
+- To install, ``cd`` to `ECLDocGenerator/`.
+- Run : ``sudo make install`` (if root permissions)
+- Else Run : ``make install`` (add default pip3 installation directory to $PATH env variable - most commonly $HOME/.local/bin)
+
+Or you can install through pip:
+- pip3 install https://github.com/lilyclemson/ECLDocGenerator
```

Comment:
A better solution is to pip install with package name as we discussed in the meeting and avoid git URL.

Reply:
we have to push to pypi for that, I can do so with my own account for now if you want